lib/elixir/lib/integer.ex

```diff
@@ -118,6 +118,44 @@ defmodule Integer do
   end
 
   @doc """
+  Runs both `div/2` and `mod/2` on the same input, and returns a tuple with both results.
+  """
+  @spec div_mod(integer, neg_integer | pos_integer) :: integer
+  def div_mod(x, y) when is_integer(x) and is_integer(y) do
```

Comment:
This is only used in Calendar.ISO module, so it should be a private function there.